waterloop-site/src/pages/Recruitment/Recruitment.tsx

```diff
@@ -43,8 +48,9 @@ const Recruitment: React.FC = () => (
         <FlexContainer>
           <h3>Our new member recruitment is now <b>closed</b> for the Winter term.</h3>
           <p>
-          Our new member recruitment for the Spring term is not open yet. Thanks for visiting and be sure to check back in the first week of May to join the team! :) Do you have prior expertise in linear induction motors, high power PCB design, or other relevant technical topics? We'd love to learn from you as an advisor; reach out to us at <a href = "mailto: contact@waterloop.ca">contact@waterloop.ca</a> if you're interested!
+          Our new member recruitment for the Spring term will be opening soon. Click below to sign up for our newsletter and be notified when postings go live. Thanks for visiting and be sure to check back in the first week of May to join the team! :) Do you have prior expertise in linear induction motors, high power PCB design, or other relevant technical topics? We'd love to learn from you as an advisor; reach out to us at <a href = "mailto: contact@waterloop.ca">contact@waterloop.ca</a> if you're interested!
```

Comment:
another minor copy update:

can we change the copy a bit:

Click below to sign up for our newsletter and be notified when postings go live! Do you have prior expertise in linear induction motors, high power PCB design, or other relevant technical topics? We'd love to learn from you as an advisor; reach out to us at contact@waterloop.ca if you're interested!